README.md

```diff
@@ -5,11 +5,11 @@
 
 Redux Clerk handles the async CRUD in your Redux App.
 
-* Provides a set of action creators for both async and synchronous actions.
-* Provides an extendable reducer.
-* Handles derived datasets and provides selectors for computing derived data.
+* Provides a [set of action creators](docs/ActionsCreators.md) for both async and synchronous actions.
```

Comment:
I think we should use either "async and sync" or "asynchronous and synchronous" here to be consistent.
Just a thought, no need to change if you don't think we should.